libraries/AP_Networking/AP_Networking_Backend.cpp

```diff
@@ -24,4 +30,36 @@ bool AP_Networking_Backend::add_route(uint8_t iface_idx, uint32_t dest_ip, uint8
     return false;
 }
 
+#if AP_NETWORKING_CAPTURE_ENABLED
+
+/*
+  write a header for the start of a new capture packet
+  capture.sem should be taken before the header is output
+  and held for the whole packet
+  note that len should include the direction byte for PPP data
+
+  caller must have checked that capture.fd != -1
+*/
+void AP_Networking_Backend::capture_header(int fd, uint32_t len)
+{
+    uint64_t utc_usec;
+    if (!AP::rtc().get_utc_usec(utc_usec)) {
```

Comment:
missing header guards

```suggestion
#if AP_RTC_ENABLED
    if (!AP::rtc().get_utc_usec(utc_usec)) {
```